client/src/environments/environment.ts

```diff
@@ -0,0 +1,17 @@
+// This file can be replaced during build by using the `fileReplacements` array.
```

Comment:
duplicating code, not good for future maintenance

Reply:
Part of the GUI